app/controllers/katello/api/v2/content_view_filter_rules_controller.rb

```diff
@@ -122,5 +119,28 @@ def process_errata_ids(select_all_params)
         []
       end
     end
+
+    private def create_rules
```

Comment:
it makes so you don't have to guess where in the file you are to know the visibility which i like